### PR TITLE
Add an advanced page for the Brain Board

### PR DESCRIPTION
**Pull request by trickeydan**
~This page is intentionally hidden from the sidebar menu. We will send a link to Students if they need the information. This is to avoid confusing students who don't want to hack around with the kit.~

### CODE REVIEW

**Comment by sedders123** [comment]
I don't think this page should be hidden. It has a warning saying we give limited support and the word 'Advanced' in the title, which makes it fairly clear this is not necessary information they must know to compete in SR. 

How do you see this page confusing students? Highly likely I could be missing something but in my mind if they don't want to hack around with the lower levels of the kit they would just ignore this page.

**Comment by trickeydan** [comment]
> I don't think this page should be hidden.

The content here is essentially a centralisation of what we send to students on a case by case basis. 

> How do you see this page confusing students? Highly likely I could be missing something but in my mind if they don't want to hack around with the lower levels of the kit they would just ignore this page.

There really is a high risk of them breaking the software image. I feel like by including it in the tree the barrier might appear lower. Students that need this content ask for it anyway.

I don't know now tbh.

**Comment by sedders123** [comment]
>There really is a high risk of them breaking the software image

If they do end up breaking the image they can just reflash? We already document how to flash SD cards as this is the current update mechanism

> I feel like by including it in the tree the barrier might appear lower.

Do we not want to lower the barrier of entry to exploring the kit?

> Students that need this content ask for it anyway

This feels like confirmation bias.

**Comment by raccube** [comment]
Considering there's a warning at the top about limited support, I think we'd be fine to link it in the navigation.

File: kit/brain_board/advanced.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Brain Board - Advanced
+---
+
+# Brain Board - Advanced
+
+When connected to your Brain Board WiFi network, it is possible to access the brain board via a terminal interface. The Brain Board is running Student Robotics OS, a purpose-built Linux distribution that does not have all of the conveniences of a usual Linux system.
```
Comment (sedders123) [inline kit/brain_board/advanced.md:8]:
```suggestion
When connected to your Brain Board WiFi network, it is possible to access the Brain Board via a terminal interface. The Brain Board is running Student Robotics OS, a purpose-built Linux distribution that does not have all of the conveniences of a usual Linux system.
```

Comment (sedders123) [reply kit/brain_board/advanced.md:8]:
```suggestion
When connected to your [Brain Board WiFi](/docs/kit/wifi), it is possible to access the brain board via a terminal interface. The Brain Board is running Student Robotics OS, a purpose-built Linux distribution that does not have all of the conveniences of a usual Linux system.
```

File: kit/brain_board/advanced.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+<div class="warning">
+The following information is about advanced features for which we only provide limited support. Proceed at your own risk.
```
Comment (sedders123) [inline kit/brain_board/advanced.md:11]: Doe this make the warning more active and less passive?
```suggestion
Proceed at your own risk. We only provide limited support for these advanced features.
```

**Review by sedders123 — APPROVED** [review]
Looks good to me! A few small suggestions inline that can be taken or left :)

**Pull request merged**